packages/rfw/lib/src/flutter/core_widgets.dart

```diff
@@ -269,6 +270,15 @@ Map<String, LocalWidgetBuilder> get _coreWidgetsDefinitions => <String, LocalWid
     );
   },
 
+  'ClipRRect': (BuildContext context, DataSource source) {
+    return ClipRRect(
+      borderRadius: ArgumentDecoders.borderRadius(source, ['borderRadius']) ?? BorderRadius.zero,
+      // clipper,
```

Comment:
looks like the convention for skipped parameters in this file is to give the type as well (see line 441)